static/src/js/components/Permission/__tests__/Permission.test.jsx

```diff
@@ -383,13 +383,13 @@ describe('Permission', () => {
 
       await waitForResponse()
 
-      expect(screen.getByText('This permission grants its assigned groups access to all of its collections that have access constraint values between 1 and 100000 (or are undefined) and that have a start and end date having some overlap with the date range 2018-04-01T04:07:58Z to 2023-03-29T04:11:01Z')).toBeInTheDocument()
-      expect(screen.getByText('This permission grants its assigned groups access to granules that have access constraint values between 1 and 10000 (or are undefined) and that have a start and end date having some overlap with the date range 2018-04-01T04:07:58Z to 2023-03-29T04:11:01Z that belong to any of its collections that have access constraint values between 1 and 100000 (or are undefined) and that have a start and end date having some overlap with the date range 2018-04-01T04:07:58Z to 2023-03-29T04:11:01Z')).toBeInTheDocument()
+      expect(screen.getByText('This permission grants its assigned groups access to all of its collections that have access constraint values between 1 and 100000 (or are undefined), and, that have a start and end date having some overlap with the date range 2018-04-01T04:07:58Z to 2023-03-29T04:11:01Z')).toBeInTheDocument()
```

Comment:
This is what I meant by punctuation, you may want to revert these changes because this result is worse than the previous. Alternatively you can figure out this `, and,`.